FWCore/ParameterSet/python/Config.py

```diff
@@ -641,6 +643,11 @@ def _dumpConfigESPrefers(self, options):
         for item in self.es_prefers_().itervalues():
             result +=options.indentation()+'es_prefer '+item.targetLabel_()+' = '+item.dumpConfig(options)
         return result
+    def _dumpPythonSubProcesses(self, l, options):
+        returnValue = ''
+        for item in l:
+            returnValue +='process.subProcess = '+item.dumpPython(options)+'\n\n'
```

Comment:
No, it will not work. The test that is no longer commented out actually found this problem after being fixed. So, this function is now also fixed in the updated PR.